src/main/java/org/apache/ibatis/migration/Environment.java

```diff
@@ -52,7 +52,9 @@
     hook_before_down,
     hook_before_each_down,
     hook_after_each_down,
-    hook_after_down
+    hook_after_down,
```

Comment:
Follows the current design for hooks. Nothing new here just adding code to support "new"